gradle.properties

```diff
@@ -7,7 +7,7 @@ enabled_platforms=fabric,forge
 archives_base_name=hermes
 mod_id=hermes
 github_name=Hermes
-mod_version=1.4.0
+mod_version=1.4.0_sub0.1
```

Comment:
put this back to what it was originally

Reply:
Doh. That wasn't supposed to get into the patch. My bad.